Modules/Quotas/src/Quotas.Infrastructure/Persistence/Database/IServiceCollectionExtensions.cs

```diff
@@ -65,8 +65,8 @@ public static void AddDatabase(this IServiceCollection services, DbOptions optio
 
     public class DbOptions
     {
-        public string Provider { get; set; }
-        public string DbConnectionString { get; set; }
+        public string? Provider { get; set; }
```

Comment:
I think these should be initialized with `= null!` since `DbOptions` is instanced as empty object but is then immediately filled in with required data like so:
```csharp
    public static void AddDatabase(this IServiceCollection services, Action<DbOptions> setupOptions)
    {
        var options = new DbOptions();
        setupOptions.Invoke(options);

        services.AddDatabase(options);
    }
```

@tnotheis what do you think?

Reply:
Yeah, I agree